test-cypress/integration/help/system-information.spec.js

```diff
@@ -0,0 +1,146 @@
+            getGroupItem(2, '.server-report-help').should('not.be.visible');
+            getGroupItem(2, '.server-report-help-btn').should('be.visible').click();
+            getGroupItem(2, '.server-report-help').should('be.visible');
+            getGroupItem(2, '.server-report-help-btn').should('be.visible').click();
+            getGroupItem(2, '.server-report-help').should('not.be.visible');
+
+            // Report status is visible
+            getGroupItem(2, '.report-status').find('.report-none').should('be.visible');
+
+            // New report should be visible and enabled
+            getGroupItem(2, '.reports-toobar').find('.new-report-btn').should('be.visible')
+                .and('not.be.disabled');
+            // Download report should be visible and disabled
+            getGroupItem(2, '.reports-toobar').find('.download-report-btn').should('be.visible')
+                .and('be.disabled');
+        });
+
+        it('JVM Arguments tab', () => {
+            // JVM arguments tab can be opened
+            selectTab(2);
+            // Tab content is visible and there 14 jvm arguments listed
+            getTabContent().find('.jvm-arguments-body').closest('.tab-pane')
+                .should('have.class', 'active')
+                .find('.list-group-item').as('groups')
+                .should('have.length', 14);
+        });
+
+        it('Configuration parameters tab', () => {
+            // Configuration parameters tab can be opened
+            selectTab(3);
+            // Tab content is visible and there 14 configurations listed
+            getTabContent().find('.configuration-parameters-body').closest('.tab-pane')
+                .should('have.class', 'active')
+                .find('.list-group-item').as('groups')
+                .should('have.length', 14);
+
+            let configs = [
+                {key: 'graphdb.dist', value: '/opt/graphdb/dist'},
+                {key: 'graphdb.foreground', value: 'yes'},
+                {key: 'graphdb.home', value: '/opt/graphdb/home'},
+                {key: 'graphdb.home.conf', value: '/opt/graphdb/home'},
+                {key: 'graphdb.home.data', value: '/opt/graphdb/home/data'},
+                {key: 'graphdb.home.logs', value: '/opt/graphdb/home/logs'},
+                {key: 'graphdb.home.work', value: '/opt/graphdb/home/work'},
+                {key: 'graphdb.workbench.cors.enable', value: 'false'},
+                {key: 'graphdb.workbench.cors.origin', value: '*'},
+                {key: 'graphdb.workbench.importDirectory', value: '/opt/graphdb/home/graphdb-import/'},
+                {key: 'graphdb.workbench.maxConnections', value: '200'},
+                {key: 'graphdb.workbench.maxInMemorySize', value: '536870912'},
+                {key: 'graphdb.workbench.maxUploadSize', value: '209715200'},
+                {key: 'graphdb.workbench.serverinfo', value: 'true'}
+            ];
+
+            cy.get('@groups').each(($groupItem, index) => {
```

Comment:
Not sure if this is a good idea, to check explictly that only these configurations are available.
And more importantly, these values are determined by the start up script thus are environment specific.

Reply:
I wasn't sure as too.